src/test/skript/tests/regressions/6623-wrong parsing order for math.sk

```diff
@@ -0,0 +1,13 @@
+
+test "wrong parsing order for math in object expr":
+	spawn a zombie at spawn of world "world"
+	set {_e} to the last spawned zombie
+
+	set {_result} to health of {_e} + max health of {_e} # used to be parsed as `health of ({z} + max health of {z})`
+	assert {_result} is set with "add result not calculated"
+	delete {_result}
+
+	set {_result} to health of {_e} / max health of {_e}
```

Comment:
```suggestion
	set {_result} to health of {_e} + max health of {_e}
	assert {_result} is set with "add result not calculated"
	delete {_result}

	set {_result} to health of {_e} / max health of {_e} # used to be parsed as `health of ({z} + max health of {z})`
```
Addition worked fine previously. It's division and multiplication that didn't work

Reply:
😬 but ayham's gonna lose his PR credit now

Reply:
my creditsss you theif 😡